README.md

```diff
@@ -29,16 +29,25 @@ Start Docker Desktop.
 - To run tests:
 
   ```
-  dev/test
```

Comment:
Looks like this script was removed (it only ran `cargo test` previously). Updated the README to reflect that, but also happy to add back a script if we want one